types/package-json-validator/package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "private": true,
+    "name": "@types/package-json-validator",
+    "version": "0.6.9999",
+    "projects": [
+        "http://package-json-validator.com"
```

Comment:
This is a dead page for me; is there a github link?

Reply:
Lovely. Updated to https://github.com/TechNickAI/package.json-validator, filed https://github.com/TechNickAI/package.json-validator/issues/76, and sent https://github.com/TechNickAI/package.json-validator/pull/77.